pkg/runtime/runtime.go

```diff
@@ -1128,10 +1128,26 @@ func (a *DaprRuntime) publishMessageGRPC(msg *pubsub.NewMessage) error {
 	}
 
 	if err != nil {
-		err = fmt.Errorf("error from app while processing pub/sub event: %s", err)
+		err = fmt.Errorf("error returned from app while processing pub/sub event %v: %s", cloudEvent.ID, err)
 		log.Debug(err)
 	}
 
+	if res.GetStatus() == 0 {
+		// SUCCESS
+		return nil
+	}
+
+	if res.GetStatus() == 1 {
+		// RETRY (not logging)
+		return fmt.Errorf("error returned from app while processing pub/sub event %v. status returned: RETRY", cloudEvent.ID)
+	}
+
+	if res.GetStatus() == 2 {
+		// DROP
+		log.Errorf("non-retriable error returned from app while processing pub/sub event %v. status returned: DROP", cloudEvent.ID)
```

Comment:
This should be a debug message. Error is when the runtime threw an error, in this case it's an informational about the app state.

Reply:
It needs to be consistent with HTTP. Also, the issue is about logging with verbosity above debug. Is warning a better level here? @RicardoNiepel  ?

Reply:
We should change the HTTP API log level then, as those are per operation calls and might cause log pollution in moderate to high volumes

Reply:
As one who recently experience this I totally +100 on @yaron2 comment. Verbose, per-operation logging on stream-like pattern actually hide the root cause of the issue.

Reply:
Do we agree on logging warning instead of error on both HTTP and GRPC?

Reply:
Warning would still flood, as Info as the default level and includes Warning.

Reply:
The whole thing from the original issue was to increase verbosity. Are we making a call that we should not?